guides/v2.3/rest/bulk-endpoints.md

```diff
@@ -9,6 +9,12 @@ functional_areas:
 
 Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
 
+{: .bs-callout .bs-callout-warning}
```

Comment:
I don't think this warning is needed.  If the command doesn't work, it's likely because Bulk Operations haven't been configured, not because of RabbitMQ or cron.